src/cursor.js

```diff
@@ -51,53 +40,42 @@ function fetch(ch, path) {
 }
 
 function update(ch, change) {
-  putOnChan(ch, change);
+  putOnChan(ch, Object.freeze(change));
```

Comment:
The change object gets passed to persisters and stakeouts, so I freeze it to make sure it's not mutated by accident.